controllers/user/Notification.js

```diff
@@ -0,0 +1,106 @@
+import moment from 'moment';
+import db from '../../models/index';
+import Util from '../../helpers/Util';
+import HttpError from '../../helpers/exceptionHandler/httpError';
+
+const { Notifications } = db;
+/**
+ * @exports Notification
+ * @class Notification
+ * @description Handles all notifications in the application
+ * */
+class Notification {
+  /* eslint-disable */
+    constructor() {
+        this.message;
+        this.userId;
+    }
+
+
+    /**
+     * get all the notification of a user
+     * @async
+     * @param  {object} req - Request object
+     * @param {object} res - Response object
+     * @return {json} Returns json object
+     * @static
+     */
+    async getNotification(req, res) {
+        const { userId } = req.authData;
+        const total = await Notifications.count({ where: { userId } });
+        const pageInfo = Util.getPageInfo(req.query.page, req.query.size, total);
+        const { page, limit, offset } = pageInfo;
+        let userNotification = await Notifications.findAll({
+            limit,
+            offset,
+            where: { userId }
+        });
+
+        if (userNotification.length === 0) {
+            return res.status(200).json({ status: 'success', message: 'You do not have any notification yet' });
+        }
+        userNotification = userNotification.map((notification) => {
+            const {
+                id,
+                isRead,
+                message,
+                createdAt,
+                updatedAt
+            } = notification;
+            const currentTime = moment(Date.now());
+            if (isRead && moment(updatedAt).diff(currentTime, 'days')) {
+                Notifications.destroy({ where: { id } });
+            }
+            return {
+                id,
+                isRead,
+                message,
+                createdAt
+            };
+        });
+        return res.status(200).json({ status: 'success', userNotification, page });
+    }
+
+    /**
+     * Update notification when it has been read
+     * @async
+     * @param  {object} req - Request object
+     * @param {object} res - Response object
+     * @return {json} Returns json object
+     * @static
+     */
+    async updateNotification(req, res) {
+        const { notificationId } = req.params;
+        const { userId } = req.authData;
+        const [response] = await Notifications.update({ isRead: true }, {
+            where: { userId, id: notificationId }
+        });
+        if (response) {
```

Comment:
I have a suggestion here. You can avoid using this `if` statement since you are using the function `HttpError.throwErrorIfNull`. A simpler way of doing this would be:
```js
HttpError.throwErrorIfNull(response, 'the error message');
return res.status(200).json({...});

```

Reply:
Yeah, thanks for this suggestion. it will actually help me eliminate the use of the If statement. I will refactor the code thanks.